js_1.js

```diff
@@ -0,0 +1,36 @@
+const item_1 = 5;
+
+console.log("item_1:", item_1);
+
+const item_2 = 3;
+
+console.log("item_2:", item_2);
+
+const item_3 = item_1 + item_2;
+
+console.log("item_3:", item_3);
+
+const item_4 = "Hello World";
+
+console.log("item_4:", item_4);
+
+console.log(item_3 + item_4)
+console.log(item_3 * item_4)
+
+const item_5 = item_3;
+
+const item_6 = 15;
+let item_6_type = typeof(item_6);
+console.log("item_6 == " + item_6_type) 
+console.log("item_6_type == " + typeof(item_6_type))
+let item_7 = "item_6";
+let item_7_type = typeof(item_7);
+console.log("item_7 ==" + item_7_type)
+console.log("item_7_type == " + typeof(item_7_type))
+
+let n = 5; 
+let nn = n + "" + n;   
+let nnn = n + "" + n + "" + n;  
```

Comment:
Достаточно одной строки, чтобы получить конкатенацию
Не забываем использовать `const` где можно и нужно
```suggestion
const nnn = "" + n + n  + n;  
```